.buildkite/pipeline.yml

```diff
@@ -647,11 +647,211 @@ steps:
+          - test_extensible_dockerfile_image_arm64
+        key: "trivy-scan-arm64-extensible-dockerfile-image"
+        agents:
+          provider: k8s
+          image: "docker.elastic.co/ci-agent-images/trivy:latest"
+        command: |-
+          mkdir -p .artifacts
+          buildkite-agent artifact download '.artifacts/*.tar.gz*' .artifacts/ --step build_extensible_dockerfile_image_arm64
+          trivy --version
+          env | grep TRIVY
+          find .artifacts -type f -name '*.tar.gz*' -exec trivy image --quiet --input {} \;
+
+      # ----
+      # Extensible Dockerfile.ftest build, tests and vunlerability scan on arm64
+      # ----
+      - label: "Building arm64 Docker image from extensible Dockerfile.ftest"
+        agents:
+          provider: aws
+          instanceType: m6g.xlarge
+          imagePrefix: ci-amazonlinux-2-aarch64
+          diskSizeGb: 40
+          diskName: '/dev/xvda'
+        env:
+          ARCHITECTURE: "arm64"
+          DOCKERFILE_PATH: "Dockerfile.ftest"
+          DOCKER_IMAGE_NAME: "docker.elastic.co/ci-agent-images/elastic-connectors-extensible-dockerfile-ftest"
+          DOCKER_ARTIFACT_KEY: "elastic-connectors-extensible-dockerfile-ftest"
+        command: ".buildkite/publish/build-docker.sh"
+        key: "build_extensible_dockerfile_ftest_image_arm64"
+        artifact_paths: ".artifacts/*.tar.gz"
+      - label: "Testing arm64 image built from extensible Dockerfile.ftest"
+        agents:
+          provider: aws
+          instanceType: m6g.xlarge
+          imagePrefix: ci-amazonlinux-2-aarch64
+          diskSizeGb: 40
+          diskName: '/dev/xvda'
+        env:
+          ARCHITECTURE: "arm64"
+          DOCKERFILE_PATH: "Dockerfile.ftest"
+          DOCKER_IMAGE_NAME: "docker.elastic.co/ci-agent-images/elastic-connectors-extensible-dockerfile-ftest"
+          DOCKER_ARTIFACT_KEY: "elastic-connectors-extensible-dockerfile-ftest"
+        depends_on: "build_extensible_dockerfile_ftest_image_arm64"
+        key: "test_extensible_dockerfile_ftest_image_arm64"
+        commands:
+          - "mkdir -p .artifacts"
+          - buildkite-agent artifact download '.artifacts/*.tar.gz*' .artifacts/ --step build_extensible_dockerfile_ftest_image_arm64
+          - ".buildkite/publish/test-docker.sh"
+      - label: "Trivy Scan arm64 extensible Dockerfile.ftest image"
+        timeout_in_minutes: 10
+        depends_on:
+          - test_extensible_dockerfile_ftest_image_arm64
+        key: "trivy-scan-arm64-extensible-dockerfile-ftest-image"
+        agents:
+          provider: k8s
+          image: "docker.elastic.co/ci-agent-images/trivy:latest"
+        command: |-
+          mkdir -p .artifacts
+          buildkite-agent artifact download '.artifacts/*.tar.gz*' .artifacts/ --step build_extensible_dockerfile_ftest_image_arm64
+          trivy --version
+          env | grep TRIVY
+          find .artifacts -type f -name '*.tar.gz*' -exec trivy image --quiet --input {} \;
```

Comment:
Similarly, I don't think we are going to get a lot of value from building and testing and scanning the extensible `Dockerfile.ftest` artifact, since it's totally similar to the extensible `Dockerfile` artifact.